features/support/best_bets.rb

```diff
@@ -125,7 +125,7 @@ def check_rummager_was_sent_a_best_bet_delete(query_es_ids)
   query_es_ids.each do |id|
     # The `delete` happens twice because it is triggered when
```

Comment:
I have deleted this comment but also added a clause on here so that queries without any bets are not created in ES: https://github.com/alphagov/search-admin/pull/75/files#diff-992addae73fbaa8a5be58e0033754c8bR12